core/nodejsActionBase/runner.js

```diff
@@ -21,6 +21,9 @@
  * This file (runner.js) must currently live in root directory for nodeJsAction.
  */
 var util = require('util');
+var child_process = require('child_process');
+var fs = require('fs');
+var path = require('path');
```

Comment:
Exactly. `require` immediately goes out to disk reads the file and sticks it in memory. Lazy loading modules saves both IO time and memory usage in paths where you don't call the `require`.